src/app/play-list/play-list.component.css

```diff
@@ -0,0 +1,55 @@
+ul{
+    text-align: center;
+}
+li {
+    border: solid 2px #000;
+    float: left;
+    display: inline-block;
+    width: 242px;
+    height: 138px;
+    margin: 3px;
+
+}
+li a{
+    display: block;
+}
+li img {
+    width: 242px;
+    height: 138px;
+}
+.video1 {
```

Comment:
Nice to have:
De youtube search API gebruiken en de HTML, CSS hierop aanpassen.